test/boost/cdc_test.cc

```diff
@@ -492,25 +507,30 @@ SEASTAR_THREAD_TEST_CASE(test_pre_image_logging) {
     }, mk_cdc_test_config()).get();
 }
 
-SEASTAR_THREAD_TEST_CASE(test_pre_image_logging_static_row) {
+SEASTAR_THREAD_TEST_CASE(test_pre_post_image_logging_static_row) {
     do_with_cql_env_thread([](cql_test_env& e) {
         auto test = [&e] (bool enabled, bool with_ttl) {
-            cquery_nofail(e, "CREATE TABLE ks.tbl (pk int, pk2 int, ck int, ck2 int, s int STATIC, val int, PRIMARY KEY((pk, pk2), ck, ck2)) WITH cdc = {'enabled':'true', 'preimage':'"s + (enabled ? "true" : "false") + "'}");
-            cquery_nofail(e, "INSERT INTO ks.tbl(pk, pk2, s) VALUES(1, 11, 111)"s + (with_ttl ? " USING TTL 654" : ""));
+            cquery_nofail(e, format("CREATE TABLE ks.tbl (pk int, pk2 int, ck int, ck2 int, s int STATIC, s2 int STATIC, val int, PRIMARY KEY((pk, pk2), ck, ck2)) "
+                "WITH cdc = {{'enabled':'true', 'preimage':'{0}', 'postimage':'{0}'}}", enabled));
+            cquery_nofail(e, "INSERT INTO ks.tbl(pk, pk2, s, s2) VALUES(1, 11, 111, 1111)"s + (with_ttl ? " USING TTL 654" : ""));
 
             auto rows = select_log(e, "tbl");
 
             BOOST_REQUIRE(to_bytes_filtered(*rows, cdc::operation::pre_image).empty());
+            BOOST_REQUIRE(to_bytes_filtered(*rows, cdc::operation::post_image).empty() == !enabled);
 
             auto first = to_bytes_filtered(*rows, cdc::operation::update);
 
             auto s_index = column_index(*rows, cdc::log_data_column_name("s"));
+            auto s2_index = column_index(*rows, cdc::log_data_column_name("s2"));
```

Comment:
All my comments/questions for the previous test also applies to this one. I wonder if both tests could be folded into one test instead of duplicating most of the code...

Reply:
It probably could, but that is not the scope of this PR.